src/styles/gallery.scss

```diff
@@ -1,15 +1,15 @@
 .projects {
-  min-height: 80vh;
   display: grid;
-  grid-template-columns: repeat(auto-fit, minmax(600px, 1fr));
-  grid-column-gap: 3rem;
-  grid-row-gap: 5rem;
+  grid-template-columns: repeat(auto-fit, minmax(300px, 1fr));
+  grid-gap: 3rem;
+  margin-top: 2rem;
+
   .project {
-    min-height: 30vh;
```

Comment:
An image will always set the height of its container. If you need more space between those containers, use margin or padding instead of setting height. In a case where an image was smaller, you'd be left with white space here to fill the container.